msix.catalog.wad.test/NavigateTabsUITest.cs

```diff
@@ -45,41 +47,66 @@ public void Clear()
         [TestMethod]
         public void NavigateMainTabs()
         {
-            
+            Console.WriteLine("Trying to Find Home Button");
             session.FindElementByName("Home").Click();
             Thread.Sleep(100);
             WindowsElement name = session.FindElementByName("MSIX Catalog");
             Assert.IsNotNull(name);
-                    
             var storeMenuItem = session.FindElementByName("Store");
             Assert.IsNotNull(storeMenuItem, "Store Menu Item not found");
             storeMenuItem.Click();
             Thread.Sleep(200);
             WindowsElement store = session.FindElementByAccessibilityId("TitleStorePackages");
             Assert.IsNotNull(store, "Store Packages Title not found");
-
             session.FindElementByName("Sideload").Click();
             Thread.Sleep(100);
             WindowsElement sdload = session.FindElementByAccessibilityId("TitleSideloadPackages");
             Assert.IsNotNull(sdload);
-        
             session.FindElementByName("Developer").Click();
             WindowsElement dvlpr = session.FindElementByAccessibilityId("TitleDeveloperPackages");
             Assert.IsNotNull(dvlpr);
-        
             session.FindElementByName("Framework").Click();
             Thread.Sleep(100);
             WindowsElement fx = session.FindElementByAccessibilityId("TitleFrameworkPackages");
             Assert.IsNotNull(fx);
-        
             session.FindElementByName("System").Click();
             Thread.Sleep(100);
             WindowsElement sys = session.FindElementByAccessibilityId("TitleSystemPackages");
             Assert.IsNotNull(sys);
         }
 
+        [TestMethod]
+        public void InspectPackageManifest()
+        {
+            session.FindElementByName("Sideload").Click();
+            Thread.Sleep(100);
+            WindowsElement sdload = session.FindElementByAccessibilityId("TitleSideloadPackages");
+            // LeftClick on "VIEW MANIFEST" at (42,6)
+            Console.WriteLine("LeftClick on \"VIEW MANIFEST\" at (42,6)");
+            var manifest = session.FindElementByName("VIEW MANIFEST");
+            manifest.Click();
+            Thread.Sleep(10000);
+            try
+            {
+                // If system doesn't have default app to open manifest with.
+                string popupAppList = "/Pane[@Name=\"Desktop 1\"][@ClassName=\"#32769\"]/Window[@AutomationId=\"Popup Window\"][@Name=\"How do you want to open this file?\"]/List[@AutomationId=\"AppsListContainer\"]/List[@AutomationId=\"RecommendedAppsList\"]/ListItem[@Name=\"Notepad\"][@ClassName=\"AppListTileElement\"]";
```

Comment:
The app used to open the manifest is not set up in the app. We just invoke the shell. IIRC the default app to open .xml files is IE not notepad.